src/ts/replayProtection/bitFlip.ts

```diff
@@ -124,6 +128,6 @@ export class BitFlipReplayProtection extends ReplayProtectionAuthority {
    * Return address of replay protection authority
    */
   public getAddress() {
-    return "0x0000000000000000000000000000000000000000";
+    return "0x0000000000000000000000000000000000000001";
   }
 }
```

Comment:
Leaving the comment here, but it refers to the whole file. The `findEmptyBit` function can be simplified quite a lot:
```typescript
 /**
   * A simple function that returns the index of the first 0 bit in the bitmap.
   * @param bitmap Bitmap to find an empty bit.
   */
  public findEmptyBit(bitmap: BigNumber) {
    const bitmapBN = new BN(bitmap.toString());
    let t = new BN(1);
    for (let i = 0; i < 256; i++) {
      if (bitmapBN.and(t).isZero()) {
        return new BigNumber(i);
      }
      t = t.add(t);
    }
    return new BigNumber("-1");
  }
```

Also, I would change all the places where an index between 0 and 255 is expected to a `number` instead of a BigNumber (can be converted when needed); should also check the argument and throw an error if bigger than 255.

Reply:
added